scripts/network_delegation/addNetworkDelegation_load.py

```diff
@@ -0,0 +1,33 @@
+from sdk import *
+
+cfg_dev = TestConfig(fullnode_dev, 11000, 100, INTERVAL_NORMAL)
```

Comment:
I think eventually we should add a function to create the config ,by randomly picking nodes ,based on how many threads the user wants to run